tests/unit/mixins/createRenderMixin.ts

```diff
@@ -97,14 +90,14 @@ registerSuite({
 			const id = cachedRender.id;
 			assert.isTrue(called, 'statechange should have been called');
 
-			assert.include(id, createCachedRenderMixin.idBase, 'should include static idBase');
-			assert.notStrictEqual(cachedRender.id, createCachedRenderMixin.idBase, 'but shouldn\'t match exactly');
+			assert.include(id, createRenderMixin.idBase, 'should include static idBase');
+			assert.notStrictEqual(cachedRender.id, createRenderMixin.idBase, 'but shouldn\'t match exactly');
 			assert.strictEqual(id, cachedRender.id, 'should return same id');
 		},
 		'is read only'() {
-			const cachedRender = createCachedRenderMixin();
+			const cachedRender = createRenderMixin();
 			assert.throws(() => {
-				cachedRender.id = 'foo';
+				(<any> cachedRender).id = 'foo';
```

Comment:
Because `.id` is marked as `readonly` now, which it is at runtime, but without `any`, TypeScript tries to keep you from making that mistake.